tools/libcloud/core_1100.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Could we place these config files in a specific folder e.g. `configs` or `default`?

Reply:
done.